Readme.md

```diff
@@ -78,3 +78,196 @@ force any process with names `geth` or `op-*` to shut down.
+ }
+ 
+```
+
+Next, the deployer needs to apply the actual intent:
+
+```bash
+op-deployer apply --l1-rpc-url=http://127.0.0.1:58138 --private-key=0x3ae90739336cd848513adb4a5d6cae372b64135fb4d214aa1a25948a21c7b7fd
+```
+
+As noted before, this only seems to work after the L1 safe head has advanced quite a bit, i.e. when geth has produced about 150 blocks.
+
+Now we can generate the genesis and rollup files:
+
+```bash
+op-deployer inspect rollup 42069 > rollup.json
+op-deployer inspect genesis 42069 > genesis.json
+```
+
+These are need to run `op-node` and `op-move` respectively. Some other useful addresses (e.g. all L1 deployments)
+can be generated with other `op-deployer inspect ...` subcommands.
+
+The updated commands to run the components are:
+
+```bash
+cargo r --bin op-move --features storage,op-upgrade --release -- --genesis.l2-contract-genesis genesis.json --auth.jwt-secret jwt.txt
+```
+
+Newer versions of `op-node` also need the L1 genesis to run. If `geth` node is already running, it's useful to copy the datadir
+first as `geth` doesn't permit to generate genesis from a dir that is being currently used:
+
+```bash
+cp -r l1_datadir/ /tmp/geth-copy
+geth dumpgenesis --datadir /tmp/geth-copy > l1_genesis.json
+```
+
+```bash
+op-node \
+  --l1 http://localhost:58138 \
+  --l1.beacon.ignore \
+  --l1.rpckind basic \
+  --l2 http://localhost:8551 \
+  --l2.jwt-secret jwt.txt \
+  --sequencer.enabled \
+  --sequencer.l1-confs 5 \
+  --verifier.l1-confs 4 \
+  --rollup.config rollup.json \
+  --rpc.addr 0.0.0.0 \
+  --rpc.port 8547 \
+  --p2p.disable \
+  --rpc.enable-admin \
+  --p2p.sequencer.key 0x66fe1b9d3babc69ad68fad96ac55b08f8d0ab3cb404c78538c0a9616ad9b1d3e \
+  --rollup.l1-chain-config l1_genesis.json # Only this line new
+```
+
+```bash
+op-batcher \
+  --l2-eth-rpc http://localhost:8545 \
+  --rollup-rpc  http://localhost:8547 \
+  --poll-interval 1s \
+  --sub-safety-margin 6 \
+  --num-confirmations 1 \
+  --safe-abort-nonce-too-low-count 3 \
+  --resubmission-timeout 30s \
+  --rpc.addr 0.0.0.0 \
+  --rpc.port 8548 \
+  --rpc.enable-admin \
+  --max-channel-duration 1 \
+  --private-key  0x20635cd468a1c32892bbde520be598519ae5141992b6d4fb7d7dd54126ff0511 \
+  --l1-eth-rpc http://localhost:58138 \
+  --throttle.unsafe-da-bytes-lower-threshold 0 # Only this line is new
+```
+
+The new proposer needs game factory instead of `L2OutputOracleProxy`. It can be found as such in the folder where `op-deployer`
+was run:
+
+```bash
+cat state.json | jq -r '.opChainDeployments[0].DisputeGameFactoryProxy'
+```
+
+```bash
+op-proposer \
+  --poll-interval 12s \
+  --rpc.port 8560 \
+  --rpc.enable-admin \
+  --rollup-rpc  http://localhost:8547\
+  --private-key  0x589a8c172e0a5bd75a2d55e4ef71470bbf06776e7ab17124fb94cb92163bb05d  \
+  --l1-eth-rpc http://localhost:58138 \
+  --num-confirmations 1 \
+  --allow-non-finalized true \
+  --game-factory-address $GAME_FACTORY_ADDRESS \ # This replaces L2OO
```

Comment:
For some reason we need to move `--game-factory-address` and `--game-type` arguments are above `--l1-eth-rpc` for the proposer to work.